README.md

```diff
@@ -7,20 +7,57 @@
 Free SVG icon font for popular brands. See them all on one page at <a href="https://simpleicons.org">SimpleIcons.org</a>. Contributions, corrections & requests can be made on GitHub. Started by <a href="https://twitter.com/bathtype">Dan Leech</a>.</p>
 </p>
 
-## Usage
+## Setup
+
+### From CDN
+
+```html
+<link rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/simple-icons-font@v2/font/simple-icons.min.css"
+      type="text/css">
+```
+
+### NodeJS
```

Comment:
I think we should add a note here about the fact that this limits the scope to `v2` like we do in the main repository.

```suggestion
These examples use the latest major version. This means you won't receive any updates following the next major release. You can use `@latest` instead to receive updates indefinitely. However, this will result in a icons not showing as expected if the icon is removed.

### NodeJS
```

Reply:
But only the NodeJS setup uses the latest major version with automatic updates not? Although the manual download gets the latest version, it's not updated automatically, so this warning should be inside "NodeJS" section :thinking:

Reply:
This note is intended for the CDN users. The comment isn't even correct for NPM/NodeJS users, as installing `@latest` will not give you updates indefinitely, only until the next major release (same as just `npm install simple-icon-font`). We could add a similar note to the NodeJS section as well, though I would expect a bit more knowledge in this regard from NPM users (but maybe that is unfair).

The reason, I think, we should add it here is because more often than not when I see packages offer CDN guidelines  in their docs the URL doesn't include a version, so the fact that we do include it may be unexpected to users of CDNs.

Reply:
I see. I've included changing only "if the icon is removed" by "if an icon is removed".

Reply:
Good catch, forgot that removing one icon may lead to problems with other icons as well :+1:

Unfortunately I made another mistake and right now the sentence still isn't correct:

```diff
- However, this will result in a icons not showing as expected if an icon is removed.
+ However, this will result in icons not showing as expected if an icon is removed.
```

Reply:
Can you fix this as well @mondeja?

Reply:
Well seen, it's done :+1:

Reply:
> Good catch, forgot that removing one icon may lead to problems with other icons as well 👍

Hmm actually, this isn't true right? If an icon is removed from the main project, then other icons should still show up correctly when using CSS classes even though the exact unicode value changed?

If so, then the sentence as it is currently written can be a bit misleading. Currently it can legitemately be interpreted as meaning: if any single icon is removed from the main project, then all icons in the font can get messed up. By the above reasoning this is not true, Therefore, I think we should write something like:

> However, this will result in icons not showing as expected when they are removed.

or

> However, this will result in icons not showing as expected if removed.

Reply:
What about this?

> However this may cause an icon to disappear if it has been removed in the latest version.

Reply:
Also works :+1: